CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Static Footer html/css
 - Static Header html/css
 - Pre-release Github Workflow
+- Example Footer test/stories/react.js
 
```

Comment:
can you add a note that we locked to eslint version due to the jest error we were getting?